python/hsfs/constructor/query.py

```diff
@@ -142,6 +150,24 @@ def join(
         the primary keys of the feature groups you are joining.
         Joins of one level are supported, no neted joins.
 
+        !!! example "Join two feature groups"
+            ```python 
+            fg1 = fs.get_feature_group("...")
+            fg2 = fs.get_feature_group("...")
+
+            feature_join = fg1.join(fg2)            
```

Comment:
Let's call it query, not 'feature_join'. To be consistent.

query = feature_join = fg1.select_all().join(fg2)